src/XMLGenerator.php

```diff
@@ -357,10 +357,15 @@ public function updateUser(
                 );
             }
         }
+
+        // If the email address is being changed, or if the SendEmailTo field
+        // is set to 'Self', then the new email address must be included in
+        // the request.
         $info = $userTag->addChild('Info');
-        if (!empty($user->getOldEmail())) {
+        if (!empty($user->getOldEmail()) || $user->getSendEmailTo() == 'Self') {
```

Comment:
Question, are we some where setting `sendEmailTo` incorrectly on the user? I'm going to try looking at this just a bit more before approving.

Reply:
I don't think we're setting it incorrectly or doing anything too strange. I think it's just a matter of:

* We're actually exercising the `updateUser` API now.
* In doing so we discovered a dependency between fields when using that endpoint that was not previously known or accounted for.
* So, we need to account for it, somehow.